packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -148,25 +133,25 @@ VTL syntax:
 
 You can also reference these org-level attributes, such as:
 
-* `$org.name`
-* `$org.locale`
-* `$org.subDomain`
+* `${org.name}`
+* `${org.locale}`
+* `${org.subDomain}`
 
-### Support for conditional logic
+## Use app context
 
 <ApiLifecycle access="ie" />
 
-All conditional logic that is supported by the Velocity Templating Engine, such as `if`, `elseif`, or `else` constructs and `foreach` loops, is available for you to use in your templates. See the [Velocity documentation](http://velocity.apache.org/engine/1.7/user-guide.html).
+Okta Identity Engine orgs have access to app context within emails using VTL. When an end user enters an authentication flow, Identity Engine stores the app context in the state token. The following properties are available in the app context:
```

Comment:
Just a nit - in some places we use `end user`, but in other places we use `end-user`.